docker/musicbrainz-tests/run_circleci_tests.sh

```diff
@@ -20,7 +20,7 @@ sudo -E -H -u musicbrainz cp docker/musicbrainz-tests/DBDefs.pm lib/
 
 sv_start_if_down postgresql redis
 
-sudo -E -H -u musicbrainz carton exec -- ./script/create_test_db.sh
+REPLICATION_TYPE=1 sudo -E -H -u musicbrainz carton exec -- ./script/create_test_db.sh
```

Comment:
The reason for this change isn’t obvious from the message for commit https://github.com/metabrainz/musicbrainz-server/pull/3197/commits/37a8cf3a193a6fd01a623ea22adb8a8b94f9c589

Reply:
Updated the commit message.